src/components/AddComment.jsx

```diff
@@ -0,0 +1,35 @@
+import { useContext, useState } from "react";
+import { postComment } from "../api";
+import { UserContext } from "./UserContext";
+
+const AddComment = ({article}) => {
+  const {loggedInUser, setLoggedInUser} = useContext(UserContext)
+  const [commentBody, setCommentBody] = useState('')
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    
+    postComment(article.article_id, commentBody, loggedInUser.username)
+    .then((response) => {
+      return response
+    }).then(() => {
+      location.reload()
```

Comment:
Ideally we don't want to reload - we want to instead have a loading sign whilst the post is ongoing and then the state should be updated once the post has gone through.